bricksrc/setpoint.py

```diff
@@ -27,10 +34,16 @@
             "Enthalpy_Setpoint": {
```

Comment:
@gtfierro Is Enthalpy_Setpoint intended to be used as an enthalpy to be maintained (target), or an enthalpy to enable something such as an economizer (lower/upper threshold)?

Reply:
I think I have mostly seen it as a target